libraries/networking/src/ResourceCache.cpp

```diff
@@ -809,6 +807,14 @@ void Resource::handleReplyFinished() {
         }
 
         auto data = _request->getData();
+        if (_request->getUrl().scheme() == "qrc") {
```

Comment:
I'm not entirely sure if this check is necessary. It may be perfectly safe to do it always